webapp/tsconfig-webpack.json

```diff
@@ -0,0 +1,8 @@
+{
+  "compilerOptions": {
+    "allowSyntheticDefaultImports": true,
+    "esModuleInterop": true,
+    "module": "commonjs",
+    "target": "es5"
```

Comment:
IIRC browsers these days usually support es6. Fine to leave as is though

Reply:
This is an option to use Typescript for Webpack configurations which are only used in nodejs runtime. I followed https://webpack.js.org/configuration/configuration-languages/ documentation. Anyway, es6 looks good.